### DISCUSSION TITLE
"The requested resource does not exist" when browsing tutorial

### DISCUSSION THREAD

**Discussion by ckoehler** [winnow-rs/winnow#354] [Q&A] [answered]
Starting at https://docs.rs/winnow/latest/winnow/_tutorial/chapter_0/index.html and clicking chapter 1 in the navigation bar goes to https://docs.rs/winnow/latest/winnow/_tutorial/chapter_0/chapter_1/index.html, which doesn't exist and gives me the error.

**Comment by epage** [comment, marked as answer]
This seems to be a rustdoc problem.  I created https://github.com/rust-lang/rust/issues/117290

**Comment by ckoehler** [comment]
Oh you're right, good catch! This was just the first docs project I've browsed in a few days. Thanks!
